content/docs/user-guide/deploying/kubernetes.md

```diff
@@ -12,6 +21,344 @@ pip install mlem[kubernetes]
 pip install kubernetes docker
 ```
 
+## Description
+
+Deploying to a Kubernetes cluster involves 2 main steps:
+
+1. Build the docker image and upload it to a registry.
+2. Create resources on the Kubernetes cluster -- specifically, a `namespace`, a
+   `deployment` and a `service`.
+
+Once this is done, one can use the usual workflow of
+[`mlem deployment run`](/doc/command-reference/deployment/run) to deploy on
+Kubernetes.
+
+<admon type="tip">
+
+You can use [`mlem types deployment kubernetes`](/doc/command-reference/types)
+to list all the configurable parameters.
+
+</admon>
+
+Most of the configurable parameters in the list above come with sensible
+defaults. But at the least, one needs to follow the structure given below:
+
+```cli
+$ mlem deployment run service_name --model model --env kubernetes --conf service_type=loadbalancer
```

Comment:
split into multiple lines so it's visible without scrolling

Reply:
This would require code changes + it isn't very long + some of the output is actually not printed by us, it's printed by the underlying kubernetes python client library. So I am skipping this. We can tackle later.

Reply:
Specifically, the part about

```
namespace created. status='{'conditions': None, 'phase': 'Active'}'
deployment created. status='{'available_replicas': None,
 'collision_count': None,
 'conditions': None,
 'observed_generation': None,
 'ready_replicas': None,
 'replicas': None,
 'unavailable_replicas': None,
 'updated_replicas': None}'
service created. status='{'conditions': None, 'load_balancer': {'ingress': None}}'
```

is not printed by us, it's done by the underlying client library

Reply:
Ahah, I mean 
```
$ mlem deployment run service_name \
    --model model \
    --env kubernetes \
    --conf service_type=loadbalancer
```
this should be applied for all the code blocks we have in docs, fyi
the user command must be visible without scrolling, when possible

Reply:
done